src/SiteExtensions/LoggingAggregate/src/Microsoft.AspNetCore.AzureAppServices.SiteExtension/Microsoft.AspNetCore.AzureAppServices.SiteExtension.csproj

```diff
@@ -10,7 +10,6 @@
     <PackageType>AzureSiteExtension</PackageType>
     <NoPackageAnalysis>true</NoPackageAnalysis>
     <IncludeBuildOutput>false</IncludeBuildOutput>
-    <IncludeSymbols>false</IncludeSymbols>
```

Comment:
I'm not sure if this is the right approach. There isn't any code in this package (see https://github.com/aspnet/AspNetCore/tree/master/src/SiteExtensions/LoggingAggregate/src/Microsoft.AspNetCore.AzureAppServices.SiteExtension). What error are you hitting with this package?

Reply:
This is one of the few packages containing Microsoft.Web.Xdt.Extensions.dll. How else would you suggest we get its .pdb deployed?

Reply:
Maybe I missed it, but I looked at the artifacts in this PR and didn't see Microsoft.Web.Xdt.Extensions.pdb.